PolyEngine/RenderingDevice/OpenGL/Src/Pipeline/ShadowMapPass.hpp

```diff
@@ -0,0 +1,54 @@
+#pragma once
+
+#include <Defines.hpp>
+#include <Common/GLUtils.hpp>
+#include <Proxy/GLShaderProgram.hpp>
+
+namespace Poly
+{
+	class RenderingSettingsComponent;
+	class GLRenderingDevice;
+	class GLShaderProgram;
+	struct SceneView;
+
+	Matrix GetProjectionForShadowMap(const SceneView& sceneView, int shadowmapSize);
+
+	void StablizeShadowProjection(Poly::Matrix& clipFromWorld, int shadowmapSize);
+	
+	int GetShadowMapSize(const RenderingSettingsComponent* renderingSettingsCmp);
```

Comment:
Are these functions needed outside of the class cpp file? If not then remove them from header

Reply:
Yes, these functions are used also in TiledForwardRenderer in all passes drawing geometry with shadows.